df.units.xml

```diff
@@ -2017,7 +2017,7 @@
         <int32_t name='temptation_lust' since='v0.47.01'/>
         <int32_t name='temptation_power' since='v0.47.01'/>
         <int32_t name='temptation_anger' since='v0.47.01'/>
-        <int32_t name='longterm_stress' since='v50.01'/>
+        <int32_t name='longterm_stress' since='v0.50.01'/>
```

Comment:
I think we fixed the assumption that version numbers start with 0 -- we do refer to it as 50.01 now in other places